File: C/C++/176.cpp
#include <iostream>
#include <string>
#include <algorithm>
using namespace std;

int main(){
    int N;
    cin >> N;
    for (int i = 0; i < N; i++){
        string s, t;
        cin >> s >> t;

        sort(s.begin(), s.end());
        sort(t.begin(), t.end());

        if (s == t){
            cout << "Yes" << endl;
        }else{
            cout << "No" << endl;
        }
    }
    return 0;
}

File: C/JavaScript/176.js
const fs = require("fs");
const input = fs.readFileSync(0, "utf8").trim().split("\n");

const N = parseInt(input[0], 10);

for (let i = 0; i < N; i++){
    let s = input[1 + i*2].split("");
    let t = input[2 + i*2].split("");

    s.sort()
    t.sort();

    if (s.join("")==t.join("")){
        console.log("Yes");
    }else{
        console.log("No");
    }
}